composables/getters/cartGetters.ts

```diff
@@ -0,0 +1,104 @@
+import { Cart, CartItem, Maybe, CrProductOption } from "~~/server/types/GraphQL"
+
+export const getCartItems = (cart: Cart): Maybe<CartItem>[] => cart?.items || []
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const getCartItemName = (item: CartItem) => item?.product?.name
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const getCartItemImage = (item: CartItem) => item?.product?.imageUrl
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const getCartItemPrice = (
+  item: CartItem
+): { regular: Maybe<number> | undefined; special: Maybe<number> | undefined } => {
+  return {
+    regular: item.product?.price?.price,
+    special: item.product?.price?.salePrice,
+  }
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const getCartItemQty = (item: CartItem): number => item?.quantity
+
+export const getCartItemAttributes = (
```

Comment:
For now, replace the logic inside this `getCartItemAttributes` with `item?.product?.options`

Reply:
Resolved